pkg/sloop/common/utilities.go

```diff
@@ -37,3 +37,13 @@ func Contains(stringList []string, elem string) bool {
 func GetFilePath(filePath string, fileName string) string {
 	return path.Join(filePath, fileName)
 }
+
+func Truncate(text string, width int) (string, error) {
```

Comment:
can you please add unit tests for Truncate?

Reply:
Added unit tests, doing this highlighted some bugs in Truncate that I've fixed.